src/pytest_plugins/consume/consume.py

```diff
@@ -291,16 +291,23 @@ def pytest_configure(config):  # noqa: D103
             force_flag=False,
             disable_infer_format=False,
         )
-    config.test_cases = TestCases.from_index_file(index_file)
 
-    all_forks_with_transitions = {  # type: ignore
-        fork for fork in set(get_forks()) | get_transition_forks() if not fork.ignore()
-    }
+    index = IndexFile.model_validate_json(index_file.read_text())
+    config.test_cases = index.test_cases
+
     for fixture_format in BaseFixture.formats.values():
         config.addinivalue_line(
             "markers",
             f"{fixture_format.format_name}: Tests in `{fixture_format.format_name}` format ",
         )
+
+    all_forks_with_transitions = {  # type: ignore
```

Comment:
I think the name we've had has been misleading. To me the current name implies only transition forks.
```suggestion
    # All forked defined within EEST
    all_forks = {  # type: ignore
```

Reply:
Is better